addons/settings/stringtable.xml

```diff
@@ -3,6 +3,7 @@
     <Package name="Settings">
         <Key ID="STR_cba_settings_Component">
             <English>Community Base Addons - Settings Component</English>
+            <German>Community Base Addons - Einstellungskomponente</German>
```

Comment:
This says component. But music/network just don't.
`Community Base Addons - Einstellungen` ?

Reply:
I like this more. Should I change it in English as well?